src/css/stars.css

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.stars__list {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: flex;
+  margin-right: 4px;
+
+  height: 16px;
+  width: 16px;
+
+  background-image: url(../images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin: 0;
+  background-image: url(../images/star.svg);
+}
+
+.stars__number-of-reviews {
```

Comment:
Why review has something connected with the stars?
Let's create this element for the `card` block.